packages/mml-web/src/elements/Video.ts

```diff
@@ -232,6 +240,11 @@ export class Video extends TransformableElement {
       const video = document.createElement("video");
       const material = this.mesh.material;
       const videoTexture = new THREE.VideoTexture(video);
+      if (this.props.emissive > 0) {
```

Comment:
I think we should use the same code to apply the emissiveness changes on both attribute update and when the material is initially created here (the code here ~matches `updateMaterialEmissiveIntensity`)